data/wacom-hid-52EB-pen.tablet

```diff
@@ -0,0 +1,22 @@
+# Wacom
+# Wacom HID 52EB Pen
+# WACF2200
+#
+# sysinfo.0VeQaUVnQ1
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/369
+
+[Device]
+Name=Wacom HID 52EB Pen, Wacom HID 52EB Finger
+ModelName=WACF2200
+Class=ISDV4
+DeviceMatch=i2c:056a:52eb;
+# Width=11
+# Height=6
```

Comment:
These two are required, please add whatever the closest w/h is in inches, rounded.

Reply:
Thanks for the fast response, fixed the nitpicks. 
13.3" in 16:10 should be W=11 and H=7.

Reply:
>  13.3" in 16:10 should be W=11 and H=7.

heh, can't you just measure the thing? It's in front of you, right? :)
Anyway, afaict no-one really looks at these numbers anyway, so I'm happy with 11/7.